<ui version="4.0" >
 <class>AnimatedStackedWidgetTest</class>
 <widget class="QWidget" name="AnimatedStackedWidgetTest" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QPushButton" name="redButton" >
       <property name="text" >
        <string>Red</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="blueButton" >
       <property name="text" >
        <string>Blue</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="AnimatedStackedWidget" name="stackedWidget" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="redPage" >
      <property name="styleSheet" >
       <string>background: red;</string>
      </property>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>Red panel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton" >
         <property name="text" >
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox" >
         <property name="text" >
          <string>CheckBox</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton" >
         <property name="text" >
          <string>RadioButton</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="bluePage" >
      <property name="styleSheet" >
       <string>background: blue;</string>
      </property>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QLabel" name="label_2" >
         <property name="text" >
          <string>Blue panel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2" >
         <property name="text" >
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_2" >
         <property name="text" >
          <string>CheckBox</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_2" >
         <property name="text" >
          <string>RadioButton</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AnimatedStackedWidget</class>
   <extends>QStackedWidget</extends>
   <header>animatedstackedwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
